lib/sys.py

```diff
@@ -17,13 +17,15 @@
 from __go__.os import Args, Stdin, Stdout, Stderr
 from __go__.grumpy import SysModules, MaxInt, NewFileFromFD  # pylint: disable=g-multiple-import
 from __go__.runtime import Version
+from __go__.math import MaxInt64
 
 argv = []
 for arg in Args:
   argv.append(arg)
 
 goversion = Version()
 maxint = MaxInt
+maxsize = MaxInt64 - 1
```

Comment:
I dont see how to get platform specific value from go.
It is currently using on a minor test, pull from CPython, 
I can use maxint, not much different I think.
Let me know If you want me to remove it.